evap/staff/importers.py

```diff
@@ -48,7 +48,7 @@ def store_in_database(self):
                                                                  'title': self.title,
                                                                  'is_active': True})
         if user.needs_login_key:
-            user.refresh_login_key()
+            user.reset_login_key_validity()
```

Comment:
isn't that exactly the wrong method in this case?

Reply:
Hmm. Actually, I don't think the call makes any sense here - login keys should be generated when needed, right?

I thought this was so one could also import login keys and they will be valid from the date imported on, but that doesn't happen, so yeah, I'd remove it. What do you and @janno42 think?
Do we want a test for this?

Reply:
Alright, @karyon, @janno42 and me agreed that we don't need this here. Will remove.